notebooks/community/ml_ops/stage7/model_monitoring_setup.ipynb

```diff
@@ -502,6 +502,84 @@
         "- For the CSV data source, grant both service accounts the \"Storage Object Viewer\" role."
```

Comment:
<p>Why is the bucket needed? Can you explain it in 1-2 sentences here so the users knows where it's gonna be used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
done